mocker/index.js

```diff
@@ -6,9 +6,10 @@ function _sha1_id(s) {
     return "{sha1}"+hex_sha1(s);
 }
 
-const entities = require("./edugain.json");
+const entities = require("./disco.json");
```

Comment:
Takes the latest file of the metadata in the mocker that includes trust information.